libethereum/ChainParams.cpp

```diff
@@ -147,6 +180,18 @@ ChainParams ChainParams::loadGenesis(string const& _json, h256 const& _stateRoot
     return cp;
 }
 
+ChainParams ChainParams::addEIPs(AdditionalEIPs const& _eips) const
```

Comment:
If we can't update the existing ChainParams instance, should we make this a static function instead?